addons/mrp_subcontracting_purchase/__manifest__.py

```diff
@@ -4,7 +4,7 @@
 {
     'name': 'Purchase and Subcontracting Management',
     'version': '0.1',
-    'category': 'Manufacturing/Purchase',
+    'category': 'Inventory/Purchase',
```

Comment:
Seems wrong to me

Reply:
thanks for feedback , so let me revert back this or can we keep this change as currently for the mrp_subcontracting_dropshipping module the given category is "Inventory/Purchase"